File: project/src/model/Posts.ts
export class Posts {
  id: number;
  src_img: string;
  type: number;
  title: string;
  desc: string;
  id_user: number;
  created_at: string;
  updated_at: string

  constructor( id: number,
    src_img: string,
    type: number,
    title: string,
    desc: string,
    id_user: number,
    created_at: string,
    updated_at: string) {
    this.id = id;
    this.src_img = src_img;
    this.type = type;
    this.title = title;
    this.desc = desc;
    this.id_user = id_user;
    this.created_at = created_at;
    this.updated_at = updated_at;
  }
}

File: backend/model/Posts.py
from sqlalchemy import DateTime
import datetime
import main

db = main.db

class Post(db.Model):
    __tablename__ = 'tb_post'
    id = db.Column(db.Integer, primary_key=True)
    src_img = db.Column(db.String)
    id_dominio = db.Column(
        db.Integer,
        db.ForeignKey('tb_dominio.id', ondelete='CASCADE'),
        nullable=False)
    id_user = db.Column(
        db.Integer,
        db.ForeignKey('tb_user.id', ondelete='CASCADE'),
        nullable=False)
    title = db.Column(db.String)
    desc = db.Column(db.String)
    created_at = db.Column(DateTime, default=datetime.datetime.utcnow)
    updated_at = db.Column(DateTime, default=datetime.datetime.utcnow)

    def __init__(self):
        self.id = None
        self.src_img = None
        self.id_dominio = None
        self.id_user = None
        self.desc = None
        self.created_at = None
        self.updated_at = None

def create_db():
    db.create_all()

create_db()
